src/components/search/SearchResults.jsx

```diff
@@ -1,8 +1,102 @@
-const SearchResults = () => {
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import AgeIcon from '../../assets/icons/icon-age.svg';
+import LessonTypeIcon from '../../assets/icons/icon-lesson.svg';
+import ScheduleIcon from '../../assets/icons/icons-schedule.svg';
+
+const SearchResults = ({ classes, currentPage, totalPages, onPageChange }) => {
+  const navigate = useNavigate();
+  const handleClick = () => {
```

Comment:
this function always redirect to the /register page, even if user already logged in, I see that it probably out of scope of this user but make sure you have ticket for show class infirmation for registered users